parser/statementStartTokens.coffee

```diff
@@ -0,0 +1,17 @@
+class StartTokens
+  @statement = [
+    'for', 'while',                                  # Start of statement
+    @expression
```

Comment:
Because of how CoffeeScript compiles into JavaScript, this doesn't work unless `@iterable` is defined _before_ `@expression`, which must be defined _before_ `@statement`